meta-resin-common/recipes-support/resin-conf-reset/resin-conf-reset/resin-conf-reset

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -e
+
+TARGET_GROUP=resin-bind.target
+RESIN_CONF_MP=@RESIN_CONF_MP@
+
+# Takes 2 arguments (What, Where)
+populate() {
+    # Re-construct directory structure and populate files
+    if [ -f $2 ]; then
+        mkdir -p "$(dirname $1)" && cp $2 $1
+    else
+        mkdir -p $1 && find $2 -type f -exec cp {} $1 +
```

Comment:
What is this trying to find/copy? Isn't this script moving everything in `/rollback` before running populate?